test/functionalTest/harnessFunctions.sh

```diff
@@ -450,7 +450,7 @@ function accumulatorStart()
 
   accumulatorStop $port
 
-  accumulator-server.py $port /notify $bindIp 2> /tmp/accumulator_$port &
+  accumulator-server.py $port /notify $bindIp > /tmp/accumulator_${port}_stdout 2> /tmp/accumulator_$port &
```

Comment:
This was added as an intent to have some debugging from the accelerator.
I had a lot of problems adding the 'blocking' services (as I don't know python).
After finishing, I let it stay. Might be useful in the future ...